04_pig_latin/pig_latin.rb

```diff
@@ -1,49 +0,0 @@
-# frozen_string_literal: true
-
-def translate(sent)
-  sent = sent.downcase
-  vowels = %w[a e i o u]
-  words = sent.split(' ')
-  result = []
-
-  words.each_with_index do |word, i|
-    translation = ''
-    qu = false
-    if vowels.include? word[0]
-      translation = word + 'ay'
-      result.push(translation)
-    else
-      word = word.split('')
-      count = 0
-      word.each_with_index do |char, _index|
-        if vowels.include? char
-          # handle words that start with 'qu'
-          if (char == 'u') && (translation[-1] == 'q')
-            qu = true
-            translation = words[i][count + 1..words[i].length] + translation + 'uay'
-            result.push(translation)
-            next
-            end
-          break
-        else
-          # handle words with 'qu' in middle
-          if (char == 'q') && (word[i + 1] == 'u')
-            qu = true
-            translation = words[i][count + 2..words[i].length] + 'quay'
-            result.push(translation)
-            next
-          else
-            translation += char
-            end
-          count += 1
-        end
-      end
-      # translation of consonant words without qu
-      unless qu
-        translation = words[i][count..words[i].length] + translation + 'ay'
-        result.push(translation)
-      end
-    end
-  end
-  result.join(' ')
-end
```

Comment:
Solve is to delete it completely? :)) I don't see the new implementation, as understanding from the commit that it should be here.

Reply:
No :)) It was deleted in order to be remaked...